content/4_Gold/Divisibility.mdx

```diff
@@ -28,7 +28,13 @@ to start.
 	>
 		good book
 	</Resource>
-
+	<Resource
```

Comment:
Move this to the other resource block

Reply:
Also I feel like "contains nice proofs and problems" would be better than "nice proofs and problems"

Reply:
tbh i feel like the original is good

Reply:
i think "contains nice proofs and problems" would be better, but it doesn't really matter.